### DISCUSSION TITLE
Where do you find a list of all supported icons?

### DISCUSSION THREAD

**Discussion by jugmac00** [marp-team/marp#110] [Q&A] [answered]
Hi,

thank you for this awesome tool! I prepared a lighting talk with it and :crossed_fingers: maybe I can present it at the world's biggest Python conference next week.

Somewhere I saw that you can use icons via `:smile:` - where can I find a list of all supported icons?

I look especially for icons for Twitter, GitHub and email which I would use for my contact details.

Thank you!

**Comment by yhatt** [comment, marked as answer]
Check out #44. `:smile:` is the shorthand for [Unicode emoji](https://unicode.org/emoji/charts/full-emoji-list.html). You can see the full list in https://github.com/markdown-it/markdown-it-emoji/blob/2.0.0/lib/data/full.json. e.g. `:email:` :arrow_right: :email:

Please note that Unicode emoji is not providing trademark icons such as Twitter and GitHub. I'm often using [Simple Icons](https://simpleicons.org/) through [Icongram](https://icongr.am/) like following:

```markdown
![](https://icongr.am/simple/twitter.svg?colored=true) Twitter
![](https://icongr.am/simple/github.svg?colored=true) GitHub
```

**Reply by jugmac00** [reply]
Thank you so much!
